src/BgZ-extended-3-0/Test/_reference/resources/medmij-bgz-ext-laboratorytestresult-specimen-01.xml

```diff
@@ -34,10 +34,12 @@
       </div>
    </text>
    <identifier>
-      <value value="TESTMC123456-003"/>
```

Comment:
Please also change the Id in the narrative accordingly (see line 10). This also applies to the other two resources.